Source/AwsCommonRuntimeKit/mqtt/Mqtt5Packets.swift

```diff
@@ -349,6 +483,34 @@ public class SubackPacket {
         self.reasonString = reasonString
         self.userProperties = userProperties
     }
+
+    public static func convertFromNative(_ from: UnsafePointer<aws_mqtt5_packet_suback_view>?) -> SubackPacket? {
+        if let _from = from {
```

Comment:
A guard will be better here instead of nesting 20 lines of code plus again, makes more sense to make it a failable init?.